nsv13/code/modules/overmap/weapons.dm

```diff
@@ -16,12 +16,41 @@
 	damage = 20
 	impact_effect_type = /obj/effect/temp_visual/impact_effect/torpedo
 
-/obj/item/projectile/bullet/torpedo
+/obj/item/projectile/bullet/light_cannon_round
+	icon_state = "pdc"
+	name = "light cannon round"
+	damage = 10
+//	flag = "overmap_light"
```

Comment:
Why aren't we using these?

Reply:
I want to push all the damage and armour changes into P3, ie do it all at once.